web/src/narrow_state.ts

```diff
@@ -42,17 +31,19 @@ export function search_terms(): Term[] {
 }
 
 export function is_message_feed_visible(): boolean {
-    return !recent_view_util.is_visible() && !inbox_util.is_visible();
+    return filter() !== undefined;
 }
```

Comment:
This change is wrong if `filter()` is going to return `undefined` when viewing `home_msg_list`?

Reply:
This was already fixed in previous pushes to the PR.